beast/fitting/fit_metrics/kernels.py

```diff
@@ -123,7 +124,7 @@ def __init__(self, shape, h = 1.0, domain = None, norm = None):
             norm = 1.0
         self._normconst = norm
         self.domain = domain
-        if callable(shape):
+        if isinstance(shape, collections.Callable):
```

Comment:
By the way, where do we use the kernels.py file in the fitting process? Am I missing something?